foreman/foreman.spec

```diff
@@ -671,9 +669,12 @@ cp config/settings.yaml.example config/settings.yaml
 sed -i 's/:locations_enabled: false/:locations_enabled: true/' config/settings.yaml
 sed -i 's/:organizations_enabled: false/:organizations_enabled: true/' config/settings.yaml
 export BUNDLER_EXT_GROUPS="default assets"
-ln -s %{nodejs_sitelib} node_modules
+ls -la %{?_scl_prefix}/tfm/root/%{nodejs_sitelib}/
+ln -s %{?_scl_prefix}/tfm/root/%{nodejs_sitelib} node_modules
```

Comment:
This is kind of ugly since it does not rely on a singular, pre-defined macro for this location. I reached a lack of knowledge point with this change in terms of how to address it. Ideas that crossed my mind:

 * define new macro in the `tfm` package to represent this value
 * require `tfm-build` and use `%{?_scl_root}`
 * other